ffi/src/ssh.rs

```diff
@@ -149,6 +154,28 @@ pub mod ffi {
             Ok(Box::new(SshPrivateKey(key)))
         }
 
+        /// Generates a new SSH ed25519 Private Key.
+        ///
+        /// No passphrase is set if `passphrase` is empty.
+        ///
+        /// No comment is set if `comment` is empty.
+        pub fn generate_ed25519(passphrase: &str, comment: &str) -> Result<Box<SshPrivateKey>, Box<PickyError>> {
```

Comment:
Added missing ed25519 SSH key generation for C# bindings

Reply:
Question: could we remove legacy JS bindings as we have a separate wasm module?

Reply:
The ones generated by Diplomat, right? I would prefer if we could use Diplomat to generate both C# and JS, but JS required further investigation back then. We could delete it until we look further into it though.

Reply:
I'll remove generated files (`api` and `docs` directory) so it is guaranteed that we will not keep stale generated JS files in our repo source tree, and keep the manually written stuff, just to be able resume work on JS diplomat bindings when needed. Ideally, we could add generated files to git ignore (for C# too) and generate them only when needed (maybe add CI step?)

Reply:
For now we are checking the generated files on purpose, because:

- It can be read along with the main application code and reviewed (even briefly) to get a complete picture of how things work. (I always briefly check the generated code in PRs like this one.)
- I can see spot the differences in development environments (typically, the diplomat-tool version)
- The code is buildable and testable right after git clone and other tools like static analysis also work right away.

That being said, I think we should at least run a check to ensure that the generated code is not stale.

I’m also not against removing the generated code completely, but I don’t think it’s necessarily bad thing to have it either.